tests/prague/eip663_dupn_swapn_exchange/test_dupn.py

```diff
@@ -0,0 +1,57 @@
+"""
+abstract: Tests [EIP-663: SWAPN, DUPN and EXCHANGE instructions](https://eips.ethereum.org/EIPS/eip-663)
+    Tests for the DUPN instruction.
+"""  # noqa: E501
+
+import pytest
+
+from ethereum_test_tools import Account, Environment, StateTestFiller, TestAddress, Transaction
+from ethereum_test_tools.eof.v1 import Container, Section
+from ethereum_test_tools.eof.v1.constants import NON_RETURNING_SECTION
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from ..eip3540_eof_v1.spec import EOF_FORK_NAME
```

Comment:
Linter is complaining about word `eip3540`, you can add this too to the `whitelist.txt`.